.github/workflows/test-inputs.yml

```diff
@@ -190,3 +190,90 @@ jobs:
         run: |
           echo "Error! The action should have failed because of the breaking change, but it has not."
           exit 1
+
+  test-toolchain-default:
+    name: Test whether action does not override the toolchain by default
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the test repository
+        uses: actions/checkout@v3
+        with:
+          repository: mgr0dzicki/cargo-semver-action-ref-slice
+          ref: patch_change
+      - name: Checkout the action
+        uses: actions/checkout@v3
+        with:
+          path: action
+      - name: Install latest nightly
+        uses: actions-rs/toolchain@v1
```

Comment:
This action is deprecated btw, and I think it'll stop working in a month or so with the new GitHub Actions API changes.

Let's switch to this one: https://github.com/obi1kenobi/cargo-semver-checks/blob/main/.github/workflows/ci.yml#L111